nvflare/private/fed/server/run_manager.py

```diff
@@ -107,5 +110,36 @@ def fire_event(self, event_type: str, fl_ctx: FLContext):
     def add_handler(self, handler: FLComponent):
         self.handlers.append(handler)
 
+    def get_cell(self):
+        return self.cell
+
+    def validate_clients(self, client_names: List[str]) -> Tuple[List[Client], List[str]]:
+        return self._get_all_clients_from_inputs(client_names)
```

Comment:
This function should be in `ServerEngine`, instead of in here.

Reply:
Refactored. Move the get_all_clients_from_inputs() to client_manager.